data/json/items/ammo/40x46mm.json

```diff
@@ -0,0 +1,153 @@
+[
+  {
+    "abstract": "40x46mm_grenade",
```

Comment:
Looks like the majority of the entries that inherit from this have `"weight": "120 g"`. Why not add it here instead?

Reply:
The factory loads all have different weights and the survivor loads should too. When we deal with projectile and propellant mass they'll get sorted out but for now I left them approximated at the same weight as M576.